_sass/custom/custom.scss

```diff
@@ -6,3 +6,122 @@
 .search-input {
   font-size: $font-size-4;
 }
+
+a.nav-list-link {
+  color: $grey-dk-200 !important;
+  font-size: $font-size-4;
+  &:hover {
+    background-image: none ;
+    color: white ;
+  }
+  &.active {
+      color: white !important;
+      font-weight: 300 !important;
+  }
+}
+a.nav-list-expander {
+  color: $grey-dk-200 !important;
+  &:hover {
+    background-image: none;
+    color: white;
+  }
+  &.active {
+      color: white;
+      font-weight: 300 ;
+  }
+}
+
+ul.nav-list {
+  li {
+    &:last-child {
+      margin-bottom: 1.5rem;
+    }
+    a {
+          line-height: 1.25rem !important;
+          vertical-align: bottom;
+          display: flex !important;
+          align-items: center;
+
+    }
+    a.nav-list-expander {
+
+    }
+  }
+}
+.side-bar {
+  @include mq(lg) {
+    width: $nav-width;
+    min-width: $nav-width;
+  }
+}
+
+.site-nav {
+  padding-top: 1.5rem;
+}
+
+.site-logo {
+  width:100%;
+  height: 100%;
+  max-height: 24px;
+  background-image: url("/assets/images/logo-memo-bank-black.svg");
+  background-position: left;
+}
+
+.site-header, .main-header {
+  border-bottom: none;
+}
+
+.site-button {
+  .icon {
+    color:$grey-dk-200 !important;
+    &:hover {
+      background-image: none;
+      color: white;
+    }
+  }
+}
+
+.site-button.nav-open {
+  .icon {
+    color: white !important;
+  }
+}
+
+
+h1 {
+  font-size: 65px !important;
```

Comment:
There's a built-in variable to control the size of h1 ([$font-size-9](https://github.com/just-the-docs/just-the-docs/blob/main/_sass/support/_variables.scss#L23)). It's not super intuitive, I agree. 😅

Reply:
Also, I'll add a media query here to prevent or H1 title to be 65px tall on mobile (where that would be too much of a good thing).